mecab/src/common.h

```diff
@@ -81,13 +82,9 @@
 #define EXIT_SUCCESS 0
 #endif
 
-#ifdef _WIN32
-#ifdef __GNUC__
+#if defined(_WIN32) && (defined(__GNUC__) || defined(_MSC_VER))
```

Comment:
We need to convert path to wide characters from UTF-8 when we use `WPATH_FORCE` on both GCC and MSVC.

Reply:
@kou Fixed on 28f7de4 Does it make sense?

Reply:
Yes!